src/types/api.ts

```diff
@@ -193,6 +232,77 @@ export interface operations {
       422: unknown
     }
   }
+  /** Get filterable incoming transfers */
+  incoming_transfers: {
+    parameters: {
+      path: {
+        chainId: string
+        address: string
+      }
+      query?: {
+        execution_date__gte?: string
+        execution_date__lte?: string
+        to?: string
+        token_address?: string
+        value?: number
+      }
+    }
+    responses: {
+      200: {
+        schema: SafeIncomingTransfersResponse
+      }
+      /** Safe not found */
+      404: unknown
+      /** Safe address checksum not valid */
+      422: unknown
+    }
+  }
+  /** Get filterable module transactions */
+  module_transactions: {
+    parameters: {
+      path: {
+        chainId: string
+        address: string
+      }
+      query?: {
+        module?: string
+      }
+    }
+    responses: {
+      200: {
+        schema: SafeModuleTransactionsResponse
+      }
+      /** Safe not found */
+      404: unknown
+      /** Safe address checksum not valid */
+      422: unknown
+    }
+  }
+  /** Get filterable multisig transactions */
+  multisig_transactions: {
+    parameters: {
+      path: {
+        chainId: string
+        address: string
+      }
+      query?: {
+        execution_date__gte?: string
+        execution_date__lte?: string
```

Comment:
Same underscore question here:
Shouldn't it be:
```suggestion
        execution_date_gte?: string
        execution_date_lte?: string
```

Reply:
See my first somment.